tests/all.js

```diff
@@ -20,6 +20,7 @@ require('../lib/plugins/logs/tests/logs');
 require('../lib/plugins/remove/tests/remove');
 require('../lib/plugins/package/tests/all');
 require('../lib/plugins/slstats/tests/slstats');
+require('../lib/plugins/rollback/tests/rollback.js');
```

Comment:
Can you remove the `.js` part here? So that all requires follow the same pattern.